### PR TITLE
docs: remove duplicate style properties

### PR DESCRIPTION
**Pull request by amareshsm**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain:
   In the documentation website removed duplicate style properties.

#### What changes did you make? (Give an overview)
   In the documentation website removed duplicate style properties.

### CODE REVIEW

**Review by snitin315 — APPROVED** [review]
LGTM, thanks!

File: docs/src/assets/scss/components/search.scss [hunk 1]
```diff
@@ -112,7 +111,6 @@
 }
 
 .search .search-results__item__title {
-    font-size: var(--step-0);
```
Comment (nzakas) [inline docs/src/assets/scss/components/search.scss:115]: This is intentionally duplicate. The first is used when the browser support variables and the second is used when the browser does not support variables.

**Review by nzakas — COMMENTED** [review]
Most of these are intentional duplications: one for browsers that support CSS variables and one for browsers that do not. I think we should leave these in place for best compatibility.

File: docs/src/assets/scss/components/search.scss (hunk 1 above)
Comment (amareshsm) [reply docs/src/assets/scss/components/search.scss:115]: Added intentional properties and removed only duplicate properties & fixed typos.

**Review by nzakas — CHANGES_REQUESTED** [review]

**Comment by nzakas** [comment]
@amareshsm are you going to get back to this PR? If not, we can close it.

**Comment by snitin315** [comment]
Closing in favor of https://github.com/eslint/eslint/pull/16466

**Pull request closed without merging**